content/blog/2023-03-14-mining-ethereum-classic-with-an-ipollo-asic-through-2miners-pool/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Mining Ethereum Classic With An iPollo ASIC Through 2Miners Pool"
+date: 2023-03-14
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["mining", "guide"]
+linkImage: ./banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/GszUmy3L48M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+## Mining Ethereum Classic
+
+![iPollo, ETC, and 2miners.](./banner.png)
+
+Since the migration of Ethereum to the proof of stake consensus algorithm in September of 2015, the hashrate of Ethereum Classic (ETC) has increased enormously.
```

Comment:
Line 21: Correct date
```suggestion
Since the migration of Ethereum to the proof of stake consensus algorithm in September of 2022, the hashrate of Ethereum Classic (ETC) has increased enormously.
```